cmd/main_test.go

```diff
@@ -101,6 +101,7 @@ email:
   password: <mypassword>
 slack:
   token: <my-token>
+  username: <override-username>
```

Comment:
Can you please also add username to https://github.com/argoproj-labs/argocd-notifications/blob/master/docs/argocd-notifications-secret.yaml#L14 . That yaml file is used to generate docs site and will be visible at the following URL https://argoproj-labs.github.io/argocd-notifications/recipients/#notification-services

Reply:
Thanks comments!

I already added.
https://github.com/argoproj-labs/argocd-notifications/pull/24/files#diff-9c3178e6de2353ed90678f170bf4f204R14